docs/modules/ROOT/pages/advanced-how-tos/installing/enabling-builds.adoc

```diff
@@ -19,3 +19,51 @@ Before users can create an application in {ProductName}, you must enable build p
+
+[[docker-build-oci-ta]]pipeline-docker-build-oci-ta:: This pipeline uses `buildah` to create a container image leveraging link:https://konflux-ci.dev/architecture/ADR/0036-trusted-artifacts.html[trusted artifacts]. It also optionally creates a source images and runs some build-time tests. Information is shared between tasks using OCI artifacts instead of PVCs. EC will pass the link:https://enterprisecontract.dev/docs/ec-policies/release_policy.html#trusted_task__trusted[trusted_task.trusted] policy as long as all data used to build the artifact is generated from trusted tasks. (link:https://quay.io/repository/konflux-ci/tekton-catalog/pipeline-docker-build-oci-ta?tab=tags[quay.io link], link:https://github.com/konflux-ci/build-definitions/blob/main/pipelines/docker-build-oci-ta/README.md[parameter documentation])
+
+[[docker-build-multi-platform-oci-ta]]pipeline-docker-build-multi-platform-oci-ta:: This pipeline uses `buildah` to create a multi-platform container image leveraging link:https://konflux-ci.dev/architecture/ADR/0036-trusted-artifacts.html[trusted artifacts]. It also optionally creates a source images and runs some build-time tests. This pipeline requires that the link:https://github.com/konflux-ci/multi-platform-controller[multi platform controller] is deployed and configured on your {ProductName} instance. Information is shared between tasks using OCI artifacts instead of PVCs. EC will pass the link:https://enterprisecontract.dev/docs/ec-policies/release_policy.html#trusted_task__trusted[trusted_task.trusted] policy as long as all data used to build the artifact is generated from trusted tasks. (link:https://quay.io/repository/konflux-ci/tekton-catalog/pipeline-docker-build-multi-platform-oci-ta?tab=tags[quay.io link], link:https://github.com/konflux-ci/build-definitions/blob/main/pipelines/docker-build-multi-platform-oci-ta/README.md[parameter documentation])
+
+[[fbc-builder]]pipeline-fbc-builder:: This pipeline uses `buildah` to create a container image. It also performs validations for xref:/advanced-how-tos/building-olm.adoc#building-the-file-based-catalog[file-based catalogs]. (link:https://quay.io/repository/konflux-ci/tekton-catalog/pipeline-fbc-builder?tab=tags[quay.io link], link:https://github.com/konflux-ci/build-definitions/blob/main/pipelines/fbc-builder/README.md[parameter documentation])
+
+
+The tasks included in the pipeline definitions can be discovered using `yq` or `jq`. For example, 
+```
+$ tkn bundle list -o yaml quay.io/konflux-ci/tekton-catalog/pipeline-docker-build:devel pipeline docker-build 2>/dev/null | yq ".spec.tasks.[].name"
```

Comment:
I noticed that the yaml this commands returns is not always a proper yaml (can return more than one root object). Trying to process it with `yq` might return unexpected results. We might want to stick with json.
https://github.com/tektoncd/cli/issues/2091

Reply:
The intention was to list all of the task names in the pipeline. Are you saying that something other than that is what happens?

Reply:
I think that if there's more than one pipeline on that bundle, you'd get the tasks only for the last pipeline.

Reply:
I select for the specific pipeline of interest. I'll leave this comment as-is as we are not generating bundles with more than one pipeline for now. We can change the documentation in the future.